nautobot/dcim/tests/test_filters.py

```diff
@@ -137,12 +137,13 @@ def common_test_data(cls):
     CircuitTermination.objects.create(circuit=circuit, location=loc0, term_side="A")
     CircuitTermination.objects.create(circuit=circuit, location=loc1, term_side="Z")
 
-    manufacturers = list(Manufacturer.objects.all()[:3])
+    manufacturers = list(
+        Manufacturer.objects.filter(device_types__isnull=False, platforms__isnull=False).distinct()[:3]
+    )
     cls.manufacturers = manufacturers
 
     platforms = Platform.objects.all()[:3]
     for num, platform in enumerate(platforms):
-        platform.manufacturer = manufacturers[num]
```

Comment:
Fixes a flaky test

Reply:
We should still assign `platform.manufacturer` here right?